src/SidePanel/TransmitSetupView.jsx

```diff
@@ -58,27 +56,40 @@ const txPowerView = (
     const compatibility = fromPCA(boardType);
     const maxDbmRangeValue = compatibility.txPower.length - 1;
 
+    const range = { min: 0, max: maxDbmRangeValue };
+
     const label = {};
     label[0] = `${compatibility.txPower[0]}`;
     label[maxDbmRangeValue] = `${compatibility.txPower[maxDbmRangeValue]}`;
     return (
         <div className="app-sidepanel-component-slider">
-            <FormLabel>
-                {`TX Power [${compatibility.txPower[txPowerIdx]} dBm]`}
+            <FormLabel htmlFor="transmit-power-slider">
+                <span className="flex-fill">Transmit power</span>
+                <NumberInlineInput
+                    value={txPowerIdx}
+                    range={{ ...range }}
+                    onChange={value =>
+                        isRunning
+                            ? txPowerUpdatedAction(txPowerIdx)
+                            : txPowerUpdatedAction(value)
```

Comment:
[minor] Remove some unecessary duplication

```suggestion
txPowerUpdatedAction(isRunning ? txPowerIdx : value)
```